docs/source/tutorial_torch_ranker_agent.rst

```diff
@@ -71,31 +66,31 @@ There are several options:
 
 Possible sources of candidates:
 
-1. **batch:**
-  Use all labels in the batch as the candidate set (with all but the
-  example's label being treated as negatives).
-  Note: with this setting, the candidate set is identical for all
-  examples in a batch. This option may be undesirable if it is possible
-  for duplicate labels to occur in a batch, since the second instance of
-  the correct label will be treated as a negative.
-2. **batch-all-cands:**
-  Use all inline candidates in the batch as candidate set.
-  Note 1: This can result in a very large number of candidates.
-  Note 2: In this case we will deduplicate candidates.
-  Note 3: just like with 'batch' the candidate set is identical for all
-  examples in a batch.
-3. **inline:**
-  If each example comes with a list of possible label candidates, use those.
-  Each teacher act for the task should contain the field 'label_candidates'.
-  Note: With this setting, each example will have its own candidate set.
-4. **fixed:**
-  Use a global candidates list, provided by the user.
-  If self.fixed_candidates is not None, use a set of fixed candidates for
-  all examples.
-  Note: this setting is not recommended for training unless the
-  universe of possible candidates is very small.
-  To use this, add the path to your text file with the candidates to the
-  flag ``--fixed-candidates-path`` or ``-fcp``.
+1.|  **batch:**
+  | Use all labels in the batch as the candidate set (with all but the
```

Comment:
what do these `|` do?

Reply:
In retrospect it got rid of the warning but didn't do what I wanted it to do. After update:

![image](https://user-images.githubusercontent.com/31896/71589239-2b972780-2af2-11ea-9de1-769530c1a3e5.png)